diff --git a/backport-include/net/netlink.h b/backport-include/net/netlink.h
index 4af7363..97c7c38 100644
--- a/backport-include/net/netlink.h
+++ b/backport-include/net/netlink.h
@@ -29,29 +29,6 @@
 #define __NLA_TYPE_MAX		20
 #define NLA_TYPE_MAX		(__NLA_TYPE_MAX - 1)
 
-enum nla_policy_validation {
-	NLA_VALIDATE_NONE,
-	NLA_VALIDATE_RANGE,
-	NLA_VALIDATE_MIN,
-	NLA_VALIDATE_MAX,
-	NLA_VALIDATE_FUNCTION,
-};
-
-struct backport_nla_policy {
-	u8		type;
-	u8		validation_type;
-	u16		len;
-	union {
-		const void *validation_data;
-		struct {
-			s16 min, max;
-		};
-		int (*validate)(const struct nlattr *attr,
-				struct netlink_ext_ack *extack);
-	};
-};
-#define nla_policy backport_nla_policy
-
 #define NLA_POLICY_EXACT_LEN(_len)	{ .type = NLA_EXACT_LEN, .len = _len }
 #define NLA_POLICY_EXACT_LEN_WARN(_len)	{ .type = NLA_EXACT_LEN_WARN, \
 					  .len = _len }
