--- a/backport-include/net/netlink.h
+++ b/backport-include/net/netlink.h
@@ -29,6 +29,7 @@
 #define __NLA_TYPE_MAX		20
 #define NLA_TYPE_MAX		(__NLA_TYPE_MAX - 1)
 
+#if LINUX_VERSION_CODE == KERNEL_VERSION(4, 19, 26) || LINUX_VERSION_CODE == KERNEL_VERSION(4, 19, 139)
 enum nla_policy_validation {
 	NLA_VALIDATE_NONE,
 	NLA_VALIDATE_RANGE,
@@ -51,6 +52,7 @@
 	};
 };
 #define nla_policy backport_nla_policy
+#endif /*NH 4.19.26 NZ 4.19.139*/
 
 #define NLA_POLICY_EXACT_LEN(_len)	{ .type = NLA_EXACT_LEN, .len = _len }
 #define NLA_POLICY_EXACT_LEN_WARN(_len)	{ .type = NLA_EXACT_LEN_WARN, \
